src/components/Visualizations/Table/HeroImage.jsx

```diff
@@ -50,6 +51,15 @@ const TableHeroImage = ({
           role="presentation"
           className={styles.image}
         />
+        {leaverStatus !== undefined && leaverStatus > 1 &&
+          <img
+            src="/assets/images/dota2/disconnect_icon.png"
+            role="presentation"
+            className={styles.abandoned}
+            data-hint={strings[`leaver_status_${leaverStatus}`]}
```

Comment:
You can't apply `data-hint` on images since `img` does not support `::before` / `::after`. Also parent should have `position: relative`